### PR TITLE
Add Shivansh Yadav to the list

### PR DESCRIPTION
**Pull request by Shivansh-yadav13**
## Proposed Changes
  - added shivanshyadav.json

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/235"></a>

### CODE REVIEW

**Review by qwe123coder — APPROVED** [review]
hey, nice work 🚀 keep making pr like this :)
Congratulations on making your first pull request to our project.

We hope you enjoyed the contribution process, and encourage you to watch for issues labelled help wanted for additional opportunities to contribute.
LGTM

**Review by voidz7 — APPROVED** [review]

**Review by rohan-kulkarni-25 — APPROVED** [review]
LGTM 💥

File: public/data/shivanshyadav.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+        "url": "http://github.com/Shivansh-yadav13",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "http://twitter.com/_shivansh_13",
+        "icon": "twitter"
+      }
+    ]
+  }
```
Comment (Panquesito7) [inline public/data/shivanshyadav.json:17]: Missing an endline.
```suggestion
}

```

File: public/data/shivanshyadav.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    "avatar": "https://github.com/Shivansh-yadav13.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "http://github.com/Shivansh-yadav13",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "http://twitter.com/_shivansh_13",
```
Comment (Panquesito7) [inline public/data/shivanshyadav.json:13]:
```suggestion
        "url": "https://twitter.com/_shivansh_13",
``
`

File: public/data/shivanshyadav.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Shivansh Yadav",
+    "bio": "🦾I Like Computer Science🤖",
+    "avatar": "https://github.com/Shivansh-yadav13.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "http://github.com/Shivansh-yadav13",
```
Comment (Panquesito7) [inline public/data/shivanshyadav.json:8]:
```suggestion
        "url": "https://github.com/Shivansh-yadav13",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Pull request merged**